crossplane.yaml

```diff
@@ -22,3 +22,5 @@ spec:
       version: ">=v0.36.0"
     - provider: xpkg.upbound.io/upbound/provider-aws-iam
       version: ">=v0.36.0"
+    - function: xpkg.upbound.io/crossplane-contrib/function-patch-and-transform
+      version: ">=v0.1.4"
```

Comment:
Make sure to use the newest version, v0.2.1. There is also an official upbound function 

https://marketplace.upbound.io/functions/upbound/function-patch-and-transform/v0.2.1